include/service/entities/template_service.js

```diff
@@ -110,7 +110,12 @@ module.exports = function(pb) {
          * @property reprocess
          * @type {Boolean}
          */
-        this.reprocess = false;
+        if(pb.config.TemplateService.useReprocess){
+            this.reprocess = true;
+        }
+        else{
+            this.reprocess = false;
+        }
```

Comment:
The config setting makes me weary.  We've seen where reprocessing everything can degrade performance.  The majority of the time the values do not need to be reprocessed.